packages/site/src/content/DataTable/DataTableNotes.mdx

```diff
@@ -0,0 +1,190 @@
+selection, or fully tailored interactions, use the atomic DataTable components.
+These building blocks give you full control over layout and behavior, while
+preserving the visual consistency of the Atlantis design system. They pair well
+with libraries like [TanStack Table](https://tanstack.com/table) to handle
+complex data management.
+
+To see examples of how the atomic DataTable components can be configured, see
+[this section of storybook](..?path=/story/components-lists-and-tables-datatable-web-composable--basic).
+These are just starting points, the components are fully composable and can
+support a wide range of use cases.
+
+### Basic Structure
+
+The DataTable uses a compound component pattern with these atomic components:
+
+```tsx
+<DataTable.Container>
+  <DataTable.Actions>{/* Filter controls, search, etc. */}</DataTable.Actions>
+
+  <DataTable.Table>
+    <DataTable.Header>
+      <DataTable.HeaderCell>Name</DataTable.HeaderCell>
+      <DataTable.HeaderCell>Email</DataTable.HeaderCell>
+      <DataTable.HeaderCell>Role</DataTable.HeaderCell>
+    </DataTable.Header>
+
+    <DataTable.Body>
+      <DataTable.Row>
+        <DataTable.Cell>John Doe</DataTable.Cell>
+        <DataTable.Cell>john@example.com</DataTable.Cell>
+        <DataTable.Cell>Admin</DataTable.Cell>
+      </DataTable.Row>
+    </DataTable.Body>
+  </DataTable.Table>
+
+  <DataTable.Footer>
+    <DataTable.Pagination>
+      <DataTable.PaginationButton direction="previous" />
+      <DataTable.PaginationButton direction="next" />
+    </DataTable.Pagination>
+  </DataTable.Footer>
+</DataTable.Container>
+```
+
+### Component Customization
+
+Atomic DataTable components are flexible and composable. Each component accepts
+standard HTML attributes, including className, style, onClick, onMouseEnter, and
+more.
+
+Rows and cells support fully custom content, including other components,
+enabling rich, interactive table layouts. See an example
+[here](/?path=/story/components-lists-and-tables-datatable-web-composable--advanced-filtering).
+
+```tsx
+<DataTable.Row>
+  <DataTable.Cell>
+    <div
+      style={{
+        display: "flex",
+        alignItems: "center",
+        gap: "var(--space-small)",
+      }}
+    >
+      <Icon name="invoice" />
+      <div>
+        <Typography fontWeight="bold">Invoice #123</Typography>
+        <Text variation="subdued">For Services rendered</Text>
+      </div>
+    </div>
+  </DataTable.Cell>
+  <DataTable.Cell>
+    <StatusLabel status="warning" label="Late" />
+  </DataTable.Cell>
+  <DataTable.Cell>
+    <div style={{ textAlign: "right" }}>$2400.00</div>
+  </DataTable.Cell>
+  <DataTable.RowActions>
+    <Button icon="edit" label="Edit" />
+    <Button icon="delete" label="Delete" />
+  </DataTable.RowActions>
+</DataTable.Row>
+```
+
+### Sortable Headers
+
+For
+[sortable columns](../?path=/story/components-lists-and-tables-datatable-web-composable--sortable),
```

Comment:
Nit: I found that the `Component Customization` link worked on this branch but this one and the pagination one redirected to the main atlantis docs.

I think this is super minor but just something to note. Maybe we can do something (or find a built in way) of handling these links to make things more consistent in general

Reply:
You know what doesn't make sense at all lol - it's the opposite for me. The links were formatted inconsistently in that file and after updating, all the links in the note tab are opening a new window for me. Let me know if you see that.

https://github.com/user-attachments/assets/42689903-5e41-42c0-9c82-9dee094acd36